conjure-java-core/src/test/resources/external-long-test-service.yml

```diff
@@ -0,0 +1,20 @@
+types:
+  imports:
+    DangerousLong:
+      external:
+        java: java.lang.Long
+      safety: do-not-log
+
+services:
+  ExternalLongTestService:
+    name: External Long Test Service
+    package: com.palantir.product
+    default-auth: header
+    base-path: /external-long
+
+    endpoints:
+      testExternalLongArg:
+        http: POST /test
+        args:
+          externalLong:
+            type: DangerousLong
```

Comment:
Let's add a few more args:
* declared-safe external imported long
* external imported long without any safety information declared
* Alias of each of the existing examples